### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by ahuizxc**
1. using default device if serial number was not set
2. create realsense_examples for launch cameras/slam etc.
3. changed the default enable pointcloud to true.

### CODE REVIEW

**Comment by ahuizxc** [comment]
@xhuan28 please help to review this pr, thanks a lot :)

File: README.md [hunk 1]
```diff
@@ -79,7 +82,7 @@ source /opt/ros/dashing/setup.bash
 source ~/ros2_ws/install/local_setup.bash
 
 cd ~/ros2_ws
-ros2 run realsense_node realsense_camera_node __params:=src/ros2_intel_realsense/realsense_ros/config/d435.yaml __ns:=/d435
+ros2 run realsense_node realsense_camera_node __params:=./src/ros2_intel_realsense/realsense_ros/config/d435.yaml __ns:=/d435
```
Comment (xhuan28) [inline README.md]: why using realsense_camera_node instead of realsense_node? I suggest using the consistent name.

Comment (xhuan28) [reply README.md]: __params:=./src/ros2_intel_realsense/realsense_ros/config/d435.yaml
suggest you use this command as below:
__params:=\`ros2 pkg prefix realsense_examples\`/share/realsense_ros/config/d435.yaml

File: README.md [hunk 2]
```diff
@@ -89,10 +92,32 @@ source /opt/ros/dashing/setup.bash
 source ~/ros2_ws/install/local_setup.bash
 
 cd ~/ros2_ws
-ros2 run realsense_node realsense_camera_node __params:=src/ros2_intel_realsense/realsense_ros/config/t265.yaml __ns:=/t265
+ros2 run realsense_node realsense_camera_node __params:=./src/ros2_intel_realsense/realsense_ros/config/t265.yaml __ns:=/t265
```
Comment (xhuan28) [inline README.md]: same as above

File: README.md [hunk 3]
```diff
@@ -89,10 +92,32 @@ source /opt/ros/dashing/setup.bash
...
+source /opt/ros/dashing/setup.bash
+source ~/ros2_ws/install/local_setup.bash
+
+cd ~/ros2_ws
+## before launch all the cameras, serial number should be set in the launch file
+ros2 launch realsense_examples rs_t265_and_d400.launch.py
 ```
 
-***there are some sample launch file in ros2_intel_realsense/realsense_ros/launch/ for reference.***
+***for more usage of these launch files in ros2_intel_realsense/realsense_examples/launch, please refer to our [robot devkit](https://inte.github.io/robot_devkit) project, includes SLAM and navigation etc.***
```
Comment (xhuan28) [inline README.md]: ros2_intel_realsense/realsense_examples/launch => realsense_examples/launch

File: realsense_examples/launch/rs_camera.launch.py [hunk 4]
```diff
@@ -15,23 +15,16 @@
...
-    )
-    return launch.LaunchDescription([rgbd_container])
\ No newline at end of file
+    rgbd_node = Node(
+        package='realsense_node',
+        node_executable='realsense_camera_node',
+        node_namespace='',
+        output='screen',
+        )
+    return launch.LaunchDescription([rgbd_node])
```
Comment (xhuan28) [inline realsense_examples/launch/rs_camera.launch.py]: at a new line at the end

**Pull request merged**